### PR TITLE
[CI] Remove one more dependency to MathComp 1

### PR DESCRIPTION
**Pull request by proux01**
This was a fake dependency, QuickChick perfectly builds with MC2 but was kept on MC1 for some nasty menhir version kludge in ci-json. This copies in ci-json the menhir config already used by CompCert.

### CODE REVIEW

**Comment by proux01** [comment]
CI green, this is ready

File: dev/ci/ci-json.sh [hunk 1]
```diff
@@ -10,6 +10,6 @@ git_download json
 if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
 ( cd "${CI_BUILD_DIR}/json"
-  make
+  make MENHIRFLAGS=--coq MENHIRFLAGS+=--coq-no-version-check
```
Comment (SkySkimmer) [inline dev/ci/ci-json.sh:13]:
```suggestion
  make MENHIRFLAGS="--coq --coq-no-version-check"
```
?

Comment (proux01) [reply dev/ci/ci-json.sh]: Why not, I guess I copied it from Nix where I got into troubles with string quoting but here we should be fine.

**Comment by proux01** [comment]
@SkySkimmer CI green

**Review by SkySkimmer — APPROVED** [review]

**Comment by SkySkimmer** [comment]
@coqbot merge now

**Pull request merged**